addon/components/frost-select.js

```diff
@@ -84,6 +84,8 @@ export default Component.extend({
     onInput: PropTypes.func,
     renderTarget: PropTypes.string,
     role: PropTypes.string,
+    disableSpaceToggle: PropTypes.bool,
```

Comment:
Why is this component being modified?  If it is necessary then this change requires the semver of this PR to be a MAJOR, right? since we're changing the behavior of this component?  Or has the default behavior remained in place and only new been added, in which case it can be a MINOR?

Reply:
the default behavior is the same